### ISSUE TITLE
is this dead? doesn't seem to work anymoe

### ISSUE THREAD

**Issue by asgeirtj** [TopTierTools/obsidian-better-plugin-page#12]

**Comment by HananoshikaYomaru** [comment]
yes, it has been unmaintained for a year. I am sorry for inconvenience but I am planning to revamp this plugin again soon.

**Comment by zenminimalist** [comment]
> yes, it has been unmaintained for a year. I am sorry for inconvenience but I am planning to revamp this plugin again soon.

Yes please, we need something like this!! : )
